containers/Home/Home.tsx

```diff
@@ -1,73 +1,189 @@
+                                                    more_vert
+                                                </span>
+                                            </button>
+                                            <ViewSideBar data={headData} />
+                                        </div>
+                                    </StyledNavBar>
+                                    <StyledWrapperHead className="col-sm-12 col-lg-3">
+                                        <ViewSideBar data={headData} />
+                                    </StyledWrapperHead>
+                                </>
+                                <StyledWrapperContent className="col-sm-12 col-lg-9">
+                                    <StyleBanner className="row">
+                                        <StyledBannerContent className="col-sm-12 col-lg-12">
+                                            <StyledTitleContent>
+                                                Discoverd my Amazing Art Space!
+                                            </StyledTitleContent>
+                                            <StyledSubTitleContent>
+                                                <p>
+                                                    {' '}
+                                                    &lt;
+                                                    <span className="code">
+                                                        code
+                                                    </span>
+                                                    &gt; I build website &lt;/
+                                                    <span className="code">
+                                                        code
+                                                    </span>
+                                                    &gt;
+                                                </p>
+                                                <button
+                                                    type="button"
+                                                    className="btn"
+                                                >
+                                                    <a href="#service">
+                                                        Explore Now
+                                                    </a>
+                                                </button>
+                                            </StyledSubTitleContent>
+                                        </StyledBannerContent>
+                                    </StyleBanner>
+
+                                    <Achievement
+                                        achieves={contentData.achieves}
+                                    />
+
+                                    <StyledContainerEducation id="education">
+                                        <p>EDUCATION</p>
+                                        <StyledEducations>
```

Comment:
This component should be reworked with passing props and extract the mock data to `mock.ts`